### PR TITLE
Feature: exchange streams

### PR DESCRIPTION
**Pull request by pflueras**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 language: java
 dist: trusty
 before_install:
+- rm -rf $HOME/.m2/repository/io/vlingo
```
Comment (VaughnVernon) [inline .travis.yml]: @pflueras What will this do when run on Travis-CI server? I mean here:

https://travis-ci.org/vlingo/vlingo-platform

File: src/main/java/io/vlingo/lattice/exchange/streams/ExchangeStreamPublisher.java [hunk 2]
```diff
@@ -15,8 +15,9 @@
 
 /**
  * This class receives messages from an {@link Exchange} and streams them using {@link StreamPublisher} capabilities.
- * This class requires careful {@link PublisherConfiguration}, especially {@link Streams.OverflowPolicy} since inappropriate overflow policy
- * may fill up {@link ExchangeStreamSource} source.
+ * This class requires careful {@link PublisherConfiguration}, especially {@link Streams.OverflowPolicy} in conjunction with {@link ExchangeStreamSource} source.
+ * Low level values for {@link PublisherConfiguration#bufferSize} reduces memory usage (heap) while overflow drop rate may increase.
```
Comment (VaughnVernon) [inline src/main/java/io/vlingo/lattice/exchange/streams/ExchangeStreamPublisher.java]: *from:* Low level values *to:* Low numeric values

File: src/main/java/io/vlingo/lattice/exchange/streams/ExchangeStreamPublisher.java [hunk 3]
```diff
@@ -15,8 +15,9 @@
...
+ * On the other hand, high level values for {@link PublisherConfiguration#bufferSize} decrease overflow drop rate while increasing memory usage (heap).
```
Comment (VaughnVernon) [inline src/main/java/io/vlingo/lattice/exchange/streams/ExchangeStreamPublisher.java]: *from:* High level values *to:* High numeric values

File: .travis.yml (hunk 1 above)
Comment (pflueras) [reply .travis.yml]: This just cleans up vlingo dependencies from local maven repo in Travis CI.
However this live was added and removed during commits, therefore it's not present anymore here.

File: src/main/java/io/vlingo/lattice/exchange/streams/ExchangeStreamPublisher.java (hunk 3 above)
Comment (pflueras) [reply src/main/java/io/vlingo/lattice/exchange/streams/ExchangeStreamPublisher.java]: Updated!

File: src/main/java/io/vlingo/lattice/exchange/streams/ExchangeStreamPublisher.java (hunk 2 above)
Comment (pflueras) [reply src/main/java/io/vlingo/lattice/exchange/streams/ExchangeStreamPublisher.java]: Updated!

**Pull request merged**